Mathlib/Analysis/Convex/Slope.lean

```diff
@@ -267,6 +265,8 @@ theorem ConvexOn.secant_mono_aux3 (hf : ConvexOn 𝕜 s f) {x y z : 𝕜} (hx :
   linarith only [hf.secant_mono_aux1 hx hz hxy hyz]
 #align convex_on.secant_mono_aux3 ConvexOn.secant_mono_aux3
 
+/-- If `f : 𝕜 → 𝕜` is convex, then for any point `a` the slope of the secant line of `f` through `a`
```

Comment:
This is not fully accurate: `f` is convex on `s`, and `a` and `x` are points in `s`. Can you rephrase this doc-string accordingly, please?

Reply:
Thank you for your comment.

Actually I had the same doubt, but I decided to write it like this in order to mimic the style of the other doc-strings in the same file. I thought that the fact that everything happens in `s` had been left implicit on purpose, so that the docs are easier to read.

What do you think is better: leave everything like it is, modify only the 3 new doc-strings or modify all the doc-strings in the file?

Reply:
Good point! I had not noticed this fact about the other doc-strings. I have asked on zulip now.

Reply:
I'm personally happy with this small lie.

maintainer merge